rc/connect.kak

```diff
@@ -1,18 +1,20 @@
 declare-option -hidden str connect_path %sh(dirname "$kak_source")
 declare-option str connect_environment
+declare-option str connect_focus 0
 
 provide-module connect %{
   define-command connect-terminal -params .. -command-completion -docstring 'Connect a terminal' %{
     terminal sh -c %{
-      kak_opt_prelude=$1 kak_opt_connect_path=$2 kak_opt_connect_environment=$3 kak_session=$4 kak_client=$5 kak_client_env_SHELL=$6
+      kak_opt_prelude=$1 kak_opt_connect_path=$2 kak_opt_connect_environment=$3 kak_opt_connect_focus=$4 kak_session=$5 kak_client=$6 kak_client_env_SHELL=$7
       . "$kak_opt_connect_path/env/default.env"
       . "$kak_opt_connect_path/env/overrides.env"
       . "$kak_opt_connect_path/env/kakoune.env"
       . "$kak_opt_connect_path/env/git.env"
+      export kak_opt_connect_focus
```

Comment:
Prefer exporting variables in one of the [`env`] files.

[`env`]: https://github.com/alexherbo2/connect.kak/tree/master/rc/env

Reply:
Oops, sorry.  Missed this.  OK.

Reply:
Questions:
1. Setting a hard value in one of these files means the user can't control it based on, e.g., `filetype`, because
2. the usual `kak_opt_*` variable expansion isn't working in the files
3. isn't the idiomatic solution to passing option variables in kakoune, e.g. `%sh{}`, the `kak_opt_*` mechanism?  Why would we deviate from that?